src/nft/components/collection/CollectionSearch.tsx

```diff
@@ -19,7 +23,8 @@ export const CollectionSearch = () => {
       height="44"
       color={{ placeholder: 'textSecondary', default: 'textPrimary' }}
       value={searchByNameText}
-      placeholder={'Search by name'}
+      placeholder={isCollectionNftsLoading ? '' : 'Search by name'}
+      className={clsx(isCollectionNftsLoading && styles.filterButtonLoading)}
```

Comment:
I think this conditional should be based off NFTState loading, it's strange when I type in a collection search query and the searchbar and filter bar become unusable in their loading states

Reply:
I can switch.  Technically the filter count changes, but it is probably a better user experience this way